templates/shared_services/airlock_notifier/porter.yaml

```diff
@@ -44,13 +44,20 @@ parameters:
   - name: smtp_server_address
     env: SMTP_SERVER_ADDRESS
     type: string
+  - name: smtp_server_port
```

Comment:
I think this should also include a default

Reply:
Perfect. I'll add port 25.